libs/components/src/lib/dial-pad/dial-pad.scss

```
@@ -0,0 +1,40 @@
+@use '../../../../../dist/libs/tokens/scss/tokens.constants' as constants;
+@use '../../../../shared/src/lib/sass/mixins/border-radius' as
+	border-radius-variables;
+
+$inline-size: 230px;
+$gap: 16px;
+
+.base {
+	display: grid;
+	margin: 16px;
+	gap: $gap;
+	inline-size: $inline-size;
```

```suggestion
	inline-size: 100%;
	box-sizing: border-box;
```
I think that the inline-size needs also to be set on host.
I also think that 230px is maybe nice on desktop but maybe will be needed to be smaller or a bit wider. 
Moving the width to host will make the devs life easier